docs/capabilities.md

```diff
@@ -47,3 +47,4 @@ title: Capabilities
 ## 9.0
 * `config => attachments => allowed` - Whether the user can upload files into a chat
 * `config => attachments => folder` - User defined folder where items should be uploaded to
+* `force-mute` - "forceMute" signaling messages can be sent to mute other participants.
```

Comment:
@nickvergessen Is this pull request going to be backported? If yes, should I move this to the 8.0 section above? Or to a new 8.0.6/8.1 section?

Reply:
9.0 is fine